frontend/packages/integration-tests-cypress/tests/crud/other-routes.spec.ts

```diff
@@ -13,41 +15,120 @@ describe('Visiting other routes', () => {
+          {
+            url: '/api-resource/ns/default/core~v1~Pod/access',
+            waitFor: () => cy.get('[data-ouia-component-type$="TableRow"]').should('be.visible'),
+          },
+          {
+            url: '/k8s/cluster/user.openshift.io~v1~User',
+            waitFor: () => {},
+          },
+          {
+            url: '/k8s/ns/openshift-machine-api/machine.openshift.io~v1beta1~Machine',
+            waitFor: () => listPage.rows.shouldBeLoaded(),
+          },
+          {
+            url: '/k8s/ns/openshift-machine-api/machine.openshift.io~v1beta1~MachineSet',
+            waitFor: () => listPage.rows.shouldBeLoaded(),
+          },
+          {
+            url: '/k8s/ns/openshift-machine-api/autoscaling.openshift.io~v1beta1~MachineAutoscaler',
+            waitFor: () => cy.byTestID('empty-message').should('be.visible'),
+          },
+          {
+            url: '/k8s/ns/openshift-machine-api/machine.openshift.io~v1beta1~MachineHealthCheck',
+            waitFor: () => listPage.rows.shouldBeLoaded(),
+          },
+          {
+            url: '/k8s/cluster/machineconfiguration.openshift.io~v1~MachineConfig',
+            waitFor: () => listPage.rows.shouldBeLoaded(),
+          },
+          {
+            url: '/k8s/cluster/machineconfiguration.openshift.io~v1~MachineConfigPool',
+            waitFor: () => listPage.rows.shouldBeLoaded(),
+          },
+          {
+            url: '/k8s/all-namespaces/monitoring.coreos.com~v1~Alertmanager',
+            waitFor: () => listPage.rows.shouldBeLoaded(),
+          },
+          {
+            url: '/k8s/ns/openshift-monitoring/monitoring.coreos.com~v1~Alertmanager/main',
+            waitFor: () => {
+              detailsPage.isLoaded();
+              cy.byTestID('label-list').should('be.visible');
+            },
+          },
+          {
+            url: '/settings/cluster',
+            waitFor: () => cy.byLegacyTestID('cluster-version').should('exist'),
+          },
+          {
+            url: '/monitoring/query-browser',
+            waitFor: () => {
+              cy.byLegacyTestID('kebab-button').should('exist');
+              // loading indicator shows in metrics dropdown until loaded
+              cy.byTestID('loading-indicator').should('not.exist');
+            },
+          },
+          {
+            // Test loading search page for a kind with no static model.
+            url: '/search/all-namespaces?kind=config.openshift.io~v1~Console',
+            waitFor: () => listPage.rows.shouldBeLoaded(),
+          },
         ]
       : []),
   ];
   otherRoutes.forEach((route) => {
-    it(`successfully displays view for route: ${route}`, () => {
-      cy.visit(route);
-      // eslint-disable-next-line cypress/no-unnecessary-waiting
-      cy.wait(5000); // wait for page to load
+    it(`successfully displays view for route: ${route.url}`, () => {
+      cy.visit(route.url);
+      cy.url().should('include', `${route.url}`);
```

Comment:
Unnecessary backticks.

```suggestion
      cy.url().should('include', route.url);
```

Should it be equal to or just include?

Reply:
I "think" _include_, whole url would be `http://..`, Route is a relative route `/api-resource/ns/default...`.

Reply:
It would be better to get the path from the URL and then compare it to what we expect rather than doing a substring match. Otherwise testing `/pods/foo` then `/pods` would incorrectly pass the URL check here before it had been changed.

Reply:
Or maybe we can prepend `Cypress.config().baseUrl` to `route.url`.

We should rename `route.url` to `route.path` if it's not a full URL. That tripped me up here :)

Reply:
thanks, fixed all of these